src/commands/bot_status/bot_status.command.ts

```diff
@@ -0,0 +1,46 @@
+import { SlashCommandPipe } from '@discord-nestjs/common';
+import {
+  Command,
+  Handler,
+  IA,
+  InjectDiscordClient,
+  InteractionEvent,
+} from '@discord-nestjs/core';
+import { Injectable } from '@nestjs/common';
+import { Client, CommandInteraction } from 'discord.js';
+import { BotStatusDto } from './bot_status.params';
+
+@Command({
+  name: 'bot-status',
+  description: "Change the bot's status",
+  defaultMemberPermissions: ['Administrator'],
+  dmPermission: false,
+})
+@Injectable()
+export class BotStatusCommand {
```

Comment:
Just a question, we don't want to persist the state anywhere?
And we also don't want to set the current song as the status?

Reply:
I find the constantly updating bot status a bit annoying, maybe the current song as status can be an option? Either way it should definitely be persistent.